src/canvas/canvas_events.ts

```diff
@@ -766,25 +769,22 @@ export class Canvas extends SelectableCanvas {
    * @param {Event} e Event object fired on mouseup
    */
   __onMouseUp(e: TPointerEvent) {
-    const transform = this._currentTransform,
-      groupSelector = this._groupSelector,
-      isClick =
-        !groupSelector || (groupSelector.left === 0 && groupSelector.top === 0);
```

Comment:
i m ok changing the way we detect if we are in a click or drag state for the mouse on canvas, but it should continue to be a const isClick as before. There is no reason to generate a change in the history for the other lines.

Reply:
8b7971f